tensorboard/components/tf_ng_tensorboard/header/containers/header.component.ts

```diff
@@ -42,35 +39,25 @@ const getDisabledPlugins = createSelector(
   (plugins): UiPluginMetadata[] => plugins.filter((plugin) => !plugin.enabled)
 );
 
-const getActivePluginIndex = createSelector(
-  getUiPlugins,
-  getActivePlugin,
-  (plugins, activePlugin) => {
-    return plugins.findIndex((plugin) => plugin.id === activePlugin);
-  }
-);
-
 @Component({
   selector: 'app-header',
-  templateUrl: './header.component.html',
-  styleUrls: ['./header.component.css'],
+  template: `
+    <app-header-component
+      [activePlugins]="plugins$ | async"
+      [disabledPlugins]="disabledPlugins$ | async"
+      [selectedPlugin]="activePlugin$ | async"
+      (onPluginSelectionChanged)="onPluginSelectionChange($event)"
+    ></app-header-component>
+  `,
 })
 export class HeaderComponent {
```

Comment:
This can now be `HeaderContainer`, and the filename can be `header-container.component.ts` or something.

Also, I'm not sure that we should separate things into `./components` and `./containers` subdirectories.  That's nbd here--just saying that this is part of settling naming (etc.) conventions at some point.

Reply:
Will take an action but I want us to talk about it later too.

Also, about the directory structure: let's fix that later holistically.